generated/nidigital/nidigital/enums.py

```diff
@@ -109,35 +211,89 @@ class SoftwareTrigger(Enum):
 
 class SourceDataMapping(Enum):
     BROADCAST = 2600
+    r'''
+    Broadcasts the waveform you specify to all sites.
+    '''
     SITE_UNIQUE = 2601
+    r'''
+    Sources unique waveform data to each site.
+    '''
 
 
 class TDREndpointTermination(Enum):
     OPEN = 3600
+    r'''
+    TDR channels are connected to an open circuit.
+    '''
     SHORT_TO_GROUND = 3601
+    r'''
+    TDR channels are connected to a short to ground.
+    '''
 
 
 class TerminationMode(Enum):
     ACTIVE_LOAD = 1200
+    r'''
+    The active load provides a constant current to a commutating voltage (Vcom).
+    '''
     VTERM = 1201
+    r'''
+    The pin driver drives Vterm.
+    '''
     HIGH_Z = 1202
+    r'''
+    The pin driver is in a non-drive state (in a high-impedance state) and the active load is disabled.
+    '''
 
 
 class TimeSetEdgeType(Enum):
     DRIVE_ON = 2800
+    r'''
+    Specifies the drive on edge of the time set.
+    '''
     DRIVE_DATA = 2801
+    r'''
+    Specifies the drive data edge of the time set.
+    '''
     DRIVE_RETURN = 2802
+    r'''
+    Specifies the drive return edge of the time set.
+    '''
     DRIVE_OFF = 2803
+    r'''
+    Specifies the drive off edge of the time set.
+    '''
     COMPARE_STROBE = 2804
+    r'''
+    Specifies the compare strobe of the time set.
+    '''
     DRIVE_DATA2 = 2805
+    r'''
+    Specifies the drive data 2 edge of the time set.
+    '''
     DRIVE_RETURN2 = 2806
+    r'''
+    Specifies the drive return 2 edge of the time set.
+    '''
     COMPARE_STROBE2 = 2807
+    r'''
+    Specifies the compare strobe 2 of the time set.
+    '''
 
 
 class TriggerType(Enum):
     NONE = 1700
+    r'''
+    Disables the start trigger.
+    '''
     DIGITAL_EDGE = 1701
+    r'''
+    Digital edge trigger.
+    '''
     SOFTWARE = 1702
+    r'''
+    Software start trigger.
+    '''
 
 
 class WriteStaticPinState(Enum):
```

Comment:
Documentation missing

Reply:
Likewise, I could not find any documentation for this enumeration.

Reply:
Will be addressed in https://github.com/ni/nimi-python/pull/1552